packages/ast-utilities/package.json

```diff
@@ -68,12 +68,6 @@
   "module": "index.mjs",
   "esnext": "index.esnext",
   "exports": {
-    "./": "./",
-    ".": {
```

Comment:
What's with thinking behind the reordering of the . import? It's doesn't make any difference but I'd say the what you get when you import  `@shopify/PACKAGENAMEHERE` would be the most important export to see, and thus it deserves to be at the top of the list